src/DataTable/DropdownFilters.jsx

```diff
@@ -1,13 +1,18 @@
 import React, { useContext, useMemo } from 'react';
+import { useIntl } from 'react-intl';
 import DataTableContext from './DataTableContext';
 import { DropdownButton } from '../Dropdown';
 import useWindowSize from '../hooks/useWindowSizeHook';
 import breakpoints from '../utils/breakpoints';
+import messages from './messages';
 
 /** The first filter will be as an input, additional filters will be available in a dropdown.  */
 function DropdownFilters() {
+  const intl = useIntl();
   const { width } = useWindowSize();
-  const { columns, numBreakoutFilters } = useContext(DataTableContext);
+  const {
+    columns, numBreakoutFilters, filtersTitle, showFiltersInSidebar,
```

Comment:
I don't think this component needs to care about filters being shown in the sidebar or not. This component is already not rendered at all if `showFiltersInSidebar` is true.

https://github.com/openedx/paragon/blob/a813d75993c1b0573611e711a00454167474a22e/src/DataTable/TableControlBar.jsx#L27-L31

```suggestion
    columns, numBreakoutFilters, filtersTitle,
```

Reply:
Removed